tools/matchms/matchms_similarity_wrapper.py

```diff
@@ -12,10 +11,22 @@
 from pandas import DataFrame
 
 
+def convert_precursor_mz(spectrum):
+    """
+    Precursor m/z must be present for ModifiedCosine similarity metric
+    and it needs to be converted to float.
+    """

```

Comment:
```suggestion
    Check the presence of precursor m/z since it is needed for ModifiedCosine similarity metric. Convert to float if 
    needed, raise error if missing.
    """

```

Reply:
I believe active voice makes for a more readable code documentation. Here is a rephrasing suggestion that carries the same information but explicitly describes the steps taken inside the method.